.github/workflows/cla-check.yaml

```diff
@@ -6,23 +6,7 @@ on:
 
 jobs:
   cla-check:
-    runs-on: self-hosted
+    runs-on: ubuntu-latest
```

Comment:
nice! I'm happy we don't have to keep running this locally on self-hosted runners anymore